server/index.js

```diff
@@ -35,32 +35,70 @@ app.get('/listings', function (req, res) {
       res.sendStatus(500);
     } else {
       res.json(data);
```

Comment:
for res.json(data), you can also send a status of success along with it

Reply:
ty